frontend/src/components/Utils/time.test.js

```diff
@@ -56,3 +56,25 @@ test('other full timestamp format', () => {
   expect(actualCentral).toEqual(expectedCentral);
   expect(actualSingapore).toEqual(expectedSingapore);
 })
+
+describe('timezones for country', () => {
+  test('legit country no spaces', () => {
+    const actual = utils.timezonesForCountry("China");
+    const expected = ["Asia/Shanghai", "Asia/Urumqi"];
+    expect(new Set(actual.sort())).toEqual(new Set(expected.sort()));
+  })
+
+  test('legit country, yes spaces', () => {
+    const actual = utils.timezonesForCountry("United States of America");
+    expect(actual).toContain("America/Anchorage");
+    expect(actual).toContain("America/New York");
+    expect(actual).not.toContain("Africa/Accra");
+  })
+
+  test('not legit country (spaces and non spaces)', () => {
+    const actual = utils.timezonesForCountry("MURICA");
```

Comment:
Just as clarification, if the country doesn't exist does it just return all time zones? (nice country by the way lol)

Reply:
Yeah. so if the user doesn't put in a country then they'll get to choose from any timezone